fmdb.xcodeproj/project.pbxproj

```diff
@@ -917,7 +918,10 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_PARAMETER = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
Lines 921, 922, and 923:

Moved the deployment targets to be defined at the project level so its easier to spot compilation issues when considering a deployment target bump. IMHO: This also makes it easier to maintain, as the project's targets simply inherit their deployment target from a single version of truth.